modules/ROOT/pages/setup.adoc

```diff
@@ -3,136 +3,36 @@ ifndef::env-site,env-github[]
-
-+
-IMPORTANT: The server group you setup must use the exact name: *B2BServerGroup*. This name is case sensitive.
-+
-
-. For each environment, set up your Mule instance by https://docs.mulesoft.com/mule-runtime/4.2/runtime-installation-task[Download and Install Mule] and unzipping it.
-
-. Add the parameters mentioned in the following `wrapper.conf` file. You'll need to adjust the memory and heap size configuration numbers indicated in the following parameters for your environment. The wrapper.conf file is located at `$MULE_HOME/conf/wrapper.conf`. From there you can locate the `wrapper.conf` user conf directory.
-+
-IMPORTANT: These are example values. If you are unsure about the Mule capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.
-+
-[source,xml,linenums]
-----
-
-# Increase the MetaSpace.
-wrapper.java.additional.7=-XX:MetaspaceSize=1256m
-wrapper.java.additional.8=-XX:MaxMetaspaceSize=2256m
-
-# Increase JVM Memory
-# Initial Java Heap Size (in MB)
-wrapper.java.initmemory=4606
-# Maximum Java Heap Size (in MB)
-wrapper.java.maxmemory=4606
-
-----
-
-. Install your license.
-+
-See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
-
-. Add the server in Runtime Manager based on the instructions in https://docs.mulesoft.com/runtime-manager/servers-create#create-a-server[Create a Server].
-
-. Start Mule and verify it is active within Anypoint Runtime Manager.
-
-. Include the created servers into the server group you created earlier: *B2BServerGroup*: https://docs.mulesoft.com/runtime-manager/server-group-add[Add Servers to a Group]
-
-
-=== Setup the Payload Storage API
-
-Anypoint Partner Manager enables you to connect your own storage solution (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the transmission content such as incoming and outgoing B2B message payloads.
-To store this content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
-Follow these steps to set up the Payload Storage API:
-
-. Download these APIs from Anypoint Exchange:
-+
-https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-storage-api[Partner Manager Content Storage API]
-+
-https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-storage-service-s3[Partner Manager Content Storage API - S3 Template] (sample implementation template)
-+
-. Install the API on a runtime that can access the storage solution.
+* Required Mule vCore capacity to handle your B2B transaction workload needs.
 
-. Make a note of the API address configurations:
+If you require assistance, contact your account representative.
 
-* Host, port, and path connection
+License for the following connectors that are used in your B2B message flows:
 
-* Connection Idle and Response timeouts
+* AS2 connector
```

Comment:
AS2 connector is optional as customers may have all their trading partners connecting via AS2